include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.evr
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_evr(DnfPackage * pkg)
+    /// @return RPM package EVR (Epoch:Version-Release). If the Epoch is 0, it is omitted from the output.
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.evr
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_evr(DnfPackage * pkg)
     std::string get_evr() const;
 
-    /// Reurn nevra withou epoch when epoch is 0
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
+    /// @return RPM package NEVRA (Name-Epoch:Version-Release.Arch). If the Epoch is 0, it is omitted from the output.
+    /// @since 5.0
+    //
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
     std::string get_nevra() const;
 
-    /// Reurn always nevra with epoch
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
+    /// @return RPM package NEVRA (Name-Epoch:Version-Release.Arch). The Epoch is always present even if it is 0.
+    /// @since 5.0
+    //
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
     std::string get_full_nevra() const;
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.group
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_group(DnfPackage * pkg)
+    /// @return RPM package Group (RPMTAG_GROUP)
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.group
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_group(DnfPackage * pkg)
     std::string get_group() const;
 
-    /// If package is installed, return get_install_size(). Return get_download_size() otherwise.
-    /// @replaces dnf:dnf/package.py:attribute:Package.size
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_size(DnfPackage * pkg)
+    /// @return If package is installed, return get_install_size(). Return get_package_size() otherwise.
+    /// @since 5.0
+    //
+    // TODO(dmach): drop and use get_install_size()/get_package_size() instead?
+    // @replaces dnf:dnf/package.py:attribute:Package.size
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_size(DnfPackage * pkg)
     unsigned long long get_size() const;
 
-    /// Return size of an RPM package: <size package="..."/>
-    /// @replaces dnf:dnf/package.py:attribute:Package.downloadsize
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_downloadsize(DnfPackage * pkg)
-    unsigned long long get_download_size() const;
-
-    /// Return size of an RPM package installed on a system: <size installed="..."/>
-    /// @replaces dnf:dnf/package.py:attribute:Package.installsize
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installsize(DnfPackage * pkg)
+    /// @return File size of the RPM package
```

Comment:
It comes either from repodata or from the file system (if the package is loaded from disk - @Commandline repo)

Reply:
I'm asking in case this should be mentioned in the docstring, I kinda think it should - just because there are two sources of the data and there may e.g. be a discrepancy due to a mistake...